firefox-ios/firefox-ios-tests/Tests/XCUITests/TopTabsTest.swift

```diff
@@ -482,11 +486,15 @@ class TopTabsTest: BaseTestCase {
 fileprivate extension BaseTestCase {
     func checkNumberOfTabsExpectedToBeOpen(expectedNumberOfTabsOpen: Int) {
         navigator.goto(TabTray)
-        var numTabsOpen = userState.numTabs
-        if iPad() {
-            numTabsOpen = app.collectionViews.firstMatch.cells.count
+        if #available(iOS 16, *) {
+            var numTabsOpen = userState.numTabs
+            if iPad() {
+                numTabsOpen = app.collectionViews.firstMatch.cells.count
+            }
+            XCTAssertEqual(numTabsOpen, expectedNumberOfTabsOpen, "The number of tabs open is not correct")
+        } else {
+            // iOS 15 does not update userState.numTabs propertly
```

Comment:
I tried to get the number of tabs from the `navbar`, but the label (which is covered by the tab tray at the moment) may not be updated.